tiledb/multirange_indexing.py

```diff
@@ -7,14 +7,25 @@
 from contextvars import ContextVar
 from dataclasses import dataclass
 from numbers import Real
-from typing import Any, Dict, Iterator, List, Optional, Sequence, Tuple, Union, cast
+from typing import (
+    Any,
+    Dict,
+    Iterator,
+    List,
+    Optional,
+    Sequence,
+    Tuple,
+    Union,
+    cast,
+)
 
 import numpy as np
 
 from .cc import TileDBError
 from .dataframe_ import check_dataframe_deps
 from .libtiledb import Array, ArraySchema, Metadata, Query
 from .main import PyQuery, increment_stat, use_stats
+from .query import Query as QueryPlaceholder
```

Comment:
Obviously `QueryPlaceholder` needs to be renamed. Objections to loading `.libtiledb.Query` to `.libtiledb.QueryWrapper`?